src/core/execute.c

```diff
@@ -733,14 +733,20 @@ static int ask_for_confirmation(const char *vc, Unit *u, const char *cmdline) {
                 return 1;
         }
 
+        /* confirm_spawn might have been disabled while we were sleeping. */
+        if (manager_is_confirm_spwan_disabled(u->manager)) {
+                r = 1;
+                goto restore_stdio;
+        }
+
         e = ellipsize(cmdline, 60, 100);
         if (!e) {
                 r = log_oom();
                 goto restore_stdio;
         }
 
         for (;;) {
-                r = ask_char(&c, "yfs?ijD", "Execute %s? [y/f/s/? for help] ", e);
+                r = ask_char(&c, "yfs?ijDc", "Execute %s? [y/f/s/? for help] ", e);
```

Comment:
It was done on purpose, the help string ("[y/f/s/? for help]") is supposed to show only the main commands and shouldn't list all commands otherwise it makes it quite unreadable.